src/Types/Filtering.hs

```diff
@@ -5,31 +5,48 @@ import Control.Monad.State
 import Data.Typeable
 import Text.Printf
 import Data.List (intercalate)
+import Types.IOFormat (Example)
+
+import Test.SmallCheck.Drivers
 
 defaultTimeoutMicro = 5 * 10^4 :: Int
```

Comment:
Not sure when this changed, but now it looks like at least one "*Micro" default value is not actually in microseconds. `defaultTimeoutMicro` looks like it's 50 milliseconds. Would you mind fixing this?

Reply:
Do you think this `defaultTimeoutMicro` is too small or large?

Reply:
@mistzzt Could you confirm how can I fix this?

Reply:
In fact, these values are used as arguments of `timeout`, which takes microseconds. These values have different scales (thus 10^4 or 10^6) so I believe we should keep the _Micro_ suffix. @michaelbjames @aaronguo1996